### DISCUSSION TITLE
Support Canadian English spelling

### DISCUSSION THREAD

**Discussion by jozilla** [monkeytypegame/monkeytype#3738] [Ideas]
While there is a British English option at the moment, it would be great to also support [Canadian English](https://en.wikipedia.org/wiki/Canadian_English#Spelling), which uses 'ou' for words like _colour_ like in British English, but uses 'ize' in words like _visualize_ or 'z' in words like _organization_ like in American English.

**Comment by MaximilienGora** [comment]
I believe it already would have this suggested, it offers many languages.

**Reply by jozilla** [reply]
It's not a different language, but a regional variant of English with different spelling. Only British English is currently available.
